kubernetes/charts/edge-kubernetes/values.yaml

```diff
@@ -0,0 +1,106 @@
+# iotedged configuration
+iotedged:
+  image:
+    repository: azureiotedge/iotedged
+    tag: 0.1.0-alpha
+    pullPolicy: Always
+  service:
+    name: iotedged
+    type: ClusterIP
+  apiVersion: '2018-06-28'
+  # TCP ports on which iotedged should listen
+  ports:
+    management: 35000
+    workload: 35001
+  data:
+    targetPath: /var/lib/iotedge
+    # In order to benefit from HA, at a minimum the data location for iotedged
+    # should be backed by a persistent volume. Please setup a persistent volume
+    # and define a persistent volume claim and provide its reference below. If
+    # this is not provided then the iotedged pod will default to using an
+    # "emptyDir" volume which doesn't work for HA because its lifetime is linked
+    # to the pod's lifetime.
+    #
+    # persistentVolumeClaim:
+    #   name: <CLAIM NAME HERE>
+    #   storageClassName: <STORAGE CLASS NAME HERE>
+  ###############################################################################
+  # Certificate settings
+  ###############################################################################
+  #
+  # Configures the certificates required to operate the IoT Edge
+  # runtime as a gateway which enables external leaf devices to securely
+  # communicate with the Edge Hub. If not specified, the required certificates
+  # are auto generated for quick start scenarios which are not intended for
+  # production environments.
+  #
+  # certificates:
+  #   device_ca_cert: "<ADD PATH TO DEVICE CA CERTIFICATE HERE>"
+  #   device_ca_pk: "<ADD PATH TO DEVICE CA PRIVATE KEY HERE>"
+  #   trusted_ca_certs: "<ADD PATH TO TRUSTED CA CERTIFICATES HERE>"
+
+iotedgedProxy:
+  image:
+    repository: traefik
+    tag: v1.7-alpine
+    pullPolicy: Always
+
+# Edge Agent image configuration
+edgeAgent:
+  containerName: edgeagent
+  image:
+    repository: azureiotedge/azureiotedge-agent
```

Comment:
>azureiotedge/azureiotedge-agent [](start = 16, length = 31)

Eventually, this will need to move to the official edge agent image.

Reply:
But it is going to be a different image with k8s code in it, right?

Reply:
The plan is to have the same agent image work for k8s, just started with mode=Kuberenetes.